extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -226,20 +229,32 @@ void ConsumeWindowsEventLog::onSchedule(const std::shared_ptr<core::ProcessConte
     }
   }
 
+  std::string mode;
+  context->getProperty(OutputFormat.getName(), mode);
+
+  writeXML_ = (mode == Both || mode == XML);
+
+  writePlainText_ = (mode == Both || mode == Plaintext);
+
+  if (writeXML_) {
+    char systemDir[MAX_PATH];
+    if (GetSystemDirectory(systemDir, sizeof(systemDir))) {
```

Comment:
It is not necessary since minificpp uses Multi-Byte Character Set and macro GetSystemDirectory resolved as GetSystemDirectoryA.

Reply:
This is a common way to use any Windows API without specifying 'A' or 'W', for instance GetSystemDirectory usage in chromium:
https://chromium.googlesource.com/chromium/src/base/+/refs/heads/master/base_paths_win.cc
We should use Windows API this way it as well.

Reply:
If it consistently gives a compilation error when you try to use LPSTR in place of LPWSTR or vica versa, I can live with this, but I still don't see the benefit of using the macro version of the functions. We are specifically using either the A or the W versions, we don't have code where we would consistently use LPTSTRs, for example, so that it would work both with Unicode or ANSI, so that the macro version would be warranted.